qiita_db/support_files/patches/plugin/deblur_v1.0.2_to_v1.0.3_102917.sql

```diff
@@ -0,0 +1,41 @@
+-- With the QIIMEq2 release, the parameters for the command "Pick closed-reference
```

Comment:
The comment is from my patch (ctrl-c + ctrl-v) can you update?

Reply:
Yup